apis/v1alpha1/shared_types.go

```diff
@@ -67,6 +67,27 @@ type ContainerSelector struct {
 	ContainerNames *[]string `json:"containerNames,omitempty"`
 }
 
+// ClNetworkNamespaceSelector identifies a network namespace for network-related
+// program types in the cluster-scoped ClusterBpfApplication object.
+type ClNetworkNamespaceSelector struct {
+	// Target namespace.
+	// +optional
+	// +kubebuilder:default:=""
+	Namespace string `json:"namespace"`
```

Comment:
it sounds like there is an assumption that k8s namespace and netns strings are the same I am not sure if that is accurate, are u sure about that ?

Reply:
I'm not sure I understand you comment.  `Namespace` is the kubernetes namespace and is different from the network namespace.  If `Namespace` is set, the code limits the pods to just that k8s namespace.  It then finds the network namespace for each of the pods found.